src/js/app.spec.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @jest-environment jsdom
+ */
+import main from './app.js';
+import { addMessageToChatHistory } from './chatWindow.js';
+import { onEnterPress, onFormSubmit } from './eventHandlers.js';
+
+jest.mock('./eventHandlers.js');
+jest.mock('./chatWindow.js');
+
+describe('app.js', () => {
```

Comment:
I'd maybe rename this to something more descriptive, like "app initialization" or something. The file naming convention already implies that this is for `app.js`.